sample-docker-app/Dockerfile.arm64v8

```diff
@@ -0,0 +1,12 @@
+FROM arm64v8/node:12.19.0-buster
+WORKDIR /usr/src/app
+
+RUN apt-get update && apt-get install -yq \
+  build-essential && \
+  apt-get clean && rm -rf /var/lib/apt/lists/*
+
+COPY package*.json ./
+RUN npm install
+COPY . .
```

Comment:
won’t this overwrite `node_modules` if it exists locally?

Reply:
Yes my mistake! the COPY should overwrite from the local `node_modules`, and surprisingly the project seemed to run on the RPi even though i couldn't run `npm install` without rebuilding some dependencies (the reason for installing `build-essential`).

Reply:
But btw, did you notice the same question could be raised for the existing Dockerfile ?

Reply:
Yes, good point: @Alexander89 could you take care of that?

Reply:
@rkuhn that should be fixed now after the `.dockerignore` update.